ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/SourceBuilderTest.java

```diff
@@ -825,11 +857,14 @@ private void givenWindowedSourceStream() {
         windowInfo,
         TIMESTAMP_COLUMN,
         SOURCE_SCHEMA,
-        OptionalInt.of(SystemColumns.CURRENT_PSEUDOCOLUMN_VERSION_NUMBER)
+        OptionalInt.of(pseudoColumnVersion)
     );
   }
+  private void givenWindowedSourceStream() {
+    givenWindowedSourceStream(SystemColumns.CURRENT_PSEUDOCOLUMN_VERSION_NUMBER);
+  }
```

Comment:
This overloading strategy might be overkill, too many extra lines/complexity for not that much extra functionality? I could just make the unit tests always use `ROWPARTITION_ROWOFFSET_PSEUDOCOLUMN_VERSION` until the feature is live, then swap to using current if we want active test coverage before the version becomes current. Or I could just manually test now by temporarily changing the current pseudocolumn version, then shipping the tests using the current version, which for the infrastructure PR would remain non-functional.

Reply:
I think the overloads are fine. It's not necessary to add overloads that are unused (such as the windowed versions) but if you follow my suggestion above to add new versions of windowed tests, then this comment becomes moot since all the new overloads will be used at that point.